graphql/src/query.rs

```diff
@@ -288,6 +289,18 @@ impl Mutation {
             .await?;
         Ok(true)
     }
+
+    async fn update_source_text(
+        &self,
+        context: &Context<'_>,
+        annotation_form: AnnotationFormInput,
```

Comment:
I would probably rename this argument to `word` and the type to `AnnotatedFormUpdate` or `UpdateAnnotatedForm` (just to keep consistent with the type it's based on).